packages/aila/src/protocol/jsonPatchProtocol.ts

```diff
@@ -475,3 +802,8 @@ export function parseJsonSafely(jsonStr: string, logging: boolean = false) {
   // Return null if no valid JSON could be extracted
   return null;
 }
+
+// Handles parsing the message parts that are part of a message
+// Each row is separated by the record separator character
+// Rows can either be a valid document OR an object that looks like this:
+// {"type":"llmMessage","patches":[{...},{...}]},"prompt":{"type":"text","message":"Some message"}}
```

Comment:
Should this go at the top of the file?

Reply:
Good spot - I must have moved the method and forgot the comment